dashdotdb/schemas/swagger.yaml

```diff
@@ -7,12 +7,13 @@ info:
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
-  version: 1.0.0
+  version: 1.1.0
 servers:
 - url: /api/v1
 paths:
-  /metrics:
+  /metrics/imagemanifestvuln:
```

Comment:
Wonder if this shouldn't be the the inverse... we already have a `/imagemanifestvuln` route, I'd just add a `/imagemanifestvuln/metrics` one. Looks more natural... not sure though. No strong opinions. Wdyt?

Reply:
Same applies to the other ones, os course.

Reply:
I think I like your suggestion and it is more natural way to extend in the future.  I'll give it a try.

Reply:
I agree with @apahim suggestion :)

Reply:
Already done.